snapcraft/tests/commands/__init__.py

```diff
@@ -60,3 +67,59 @@ def mock_snap_output(command, *args, **kwargs):
                 sha3_384=get_sample_key(name)['sha3-384'])
     else:
         raise AssertionError('Unhandled command: {}'.format(command))
+
+
+class CommandBaseTestCase(tests.TestCase):
+
+    def setUp(self):
+        super().setUp()
+        self.runner = CliRunner()
+
+    def run_command(self, args, **kwargs):
+        return self.runner.invoke(run, args, env=os.environ,
```

Comment:
Why the explicit env=os.environ here? That seems like it should be the default outside of test cases already.